internal/serialization/field_description.go

```diff
@@ -16,20 +16,20 @@
 
 package serialization
 
-import pserialization "github.com/hazelcast/hazelcast-go-client/serialization"
+import pubserialization "github.com/hazelcast/hazelcast-go-client/serialization"
 
 type FieldDescriptor struct {
-	fieldName string
-	fieldKind pserialization.FieldKind
+	FieldName string
```

Comment:
maybe we can rename this to `name` in a similar fashion to `fieldKind` -> `kind` change

Reply:
That makes sense. Renamed at: https://github.com/hazelcast/hazelcast-go-client/pull/910/commits/8506b81cd1fc06f50be586facc47cc0e57fd396b